aries-site/src/pages/components/skeleton.mdx

```diff
@@ -16,15 +16,11 @@ import { DashBoardSkeletonExample, SkeletonAnatomy } from '../../examples';
 </Example>
 
 ### When to use
-Skeletons are used on all pages to provide a visual layout of the content being loaded.  They help users perceive a loading period as being shorter in duration.
+- The interface is content-rich and has asynchronous loading, in order to provide the user with a sense of the page structure and to minimize layout shifts when content is loaded in.
 
 ### When not to use
-Skeletons should not be used if:
-- The data or elements being loaded are unknown.
-- The user is awaiting feedback for in-context actions such as submitting a form or fetching search results.
-  - The Spinner component may be more appropriate.
-- The loading process takes more than a few minutes to load.
-  - Pair with a message or notification to provide the user with better visibility of a system’s status.
+- The user is awaiting feedback for in-context actions such as fetching search results. Use a [Spinner](/components/spinner) instead.
+- The user is awaiting feedback on a submission process, like submitting a form. Use a [Busy Button](/components/button#busy-button) instead.
```

Comment:
Modifying the search results use case. I'd argue that the structure of search results is known (i.e. we know the format in which the UI will present the search results) and should use skeleton. Spinners and progress bars should only be used when a process is being run by a service and there is no content being returned other than a success/error message.

I know these suggestions aren't as bullet-pointy. Feel free to play with the wording, but the criteria that should be used is:
- If content is being returned and how the UI presents that content is known, use skeleton.
- Use busy button, progress bar, or spinner to indicate a service or process is being executed and feedback is presented to users in the form of a success/error confirmation.

```suggestion
- Known, static content such as page titles, section headings, and navigational elements which are always shown and independent of data-driven content.
- The user is awaiting feedback on a submission process, like submitting a form. Use a [Busy Button](/components/button#busy-button) instead.
- The user is awaiting feedback for in-context actions such as initiating longer running processes. Instead, use a progress bar (coming soon) for determinate processes or [Spinner](/components/spinner) for indeterminate.
```

OR 

```suggestion
- Known, static content such as page titles, section headings, and navigational elements which are always shown and independent of data-driven content.
- When the purpose of the feedback is to communicate that a service or process is running and the user desires confirmation of successful completion. Instead, use a [Busy Button](/components/button#busy-button) for a submission process such as submitting a form; For longer running processes, use a progress bar (coming soon) for determinate processes or [Spinner](/components/spinner) for indeterminate.
```

OR

```suggestion
- Known, static content such as page titles, section headings, and navigational elements which are always shown and independent of data-driven content.
- A user desires feedback communicating a service or process is running and desires confirmation of successful completion. Use [busy button, progress bar, or spinner](link to the progress indicators page you are creating which will provide the guidance for when to use each) instead.
```